app/components/MazemapEmbed/index.js

```diff
@@ -32,10 +35,60 @@ export const MazemapEmbed = ({ mazemapPoi, ...props }: Props) => {
       minZoom: 10,
       maxZoom: 20,
       zLevelControl: true,
-      scrollZoom: true,
+      scrollZoom: false,
       doubleClickZoom: false,
+      dragRotate: false,
+      dragPan: false,
       touchZoomRotate: false,
+      touchPitch: false, //this is a horrible feature
+      pitchWithRotate: false,
     });
+    embeddedMazemap.dragPan._mousePan.enable();
+
+    window.addEventListener('keydown', (e) => {
```

Comment:
The window events should be removed when unmounting the component. As they are attached to the global `window`-object, they will stay there after closing the map and stack up when you open new maps.

If you return a function from the useEffect this will be run on unmount, so something like this should fix it:
```javascript
const onKeyDown = (e) => { ... }
const onKeyUp = (e) => { ... }
window.addEventListener('keydown', onKeyDown);
window.addEventListener('keyup', onKeyUp);
```
and at the end of the useEffect
```javascript
return () => {
  window.removeEventListener('keydown', onKeyDown);
  window.removeEventListener('keyup', onKeyUp);
};
```